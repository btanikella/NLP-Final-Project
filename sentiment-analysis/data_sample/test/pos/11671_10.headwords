description
Also documentary the X.
hilarious views
diversive
