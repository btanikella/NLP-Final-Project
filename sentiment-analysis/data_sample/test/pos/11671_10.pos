DT CD CC DT NN RB RB JJ NN IN DT NN NN IN DT NNP IN DT JJ CD POS
RB  DT JJ NN IN JJ NNS IN DT JJ NN CC DT NNP NNP
NNPS POS JJ NNS IN DT NN VBP RB JJ
WRB MD NN VB RB JJ IN DT NN IN VBN NN : RB CD NNS IN : VB RP
