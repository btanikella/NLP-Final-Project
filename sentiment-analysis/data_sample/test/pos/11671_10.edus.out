Sentence #1 (22 tokens):
The One and the Only!The only really good description of the punk movement in the LA in the early 80's.
[Text=The CharacterOffsetBegin=4 CharacterOffsetEnd=7 PartOfSpeech=DT] [Text=One CharacterOffsetBegin=8 CharacterOffsetEnd=11 PartOfSpeech=CD] [Text=and CharacterOffsetBegin=12 CharacterOffsetEnd=15 PartOfSpeech=CC] [Text=the CharacterOffsetBegin=16 CharacterOffsetEnd=19 PartOfSpeech=DT] [Text=Only!The CharacterOffsetBegin=20 CharacterOffsetEnd=28 PartOfSpeech=NN] [Text=only CharacterOffsetBegin=29 CharacterOffsetEnd=33 PartOfSpeech=RB] [Text=really CharacterOffsetBegin=34 CharacterOffsetEnd=40 PartOfSpeech=RB] [Text=good CharacterOffsetBegin=41 CharacterOffsetEnd=45 PartOfSpeech=JJ] [Text=description CharacterOffsetBegin=46 CharacterOffsetEnd=57 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=58 CharacterOffsetEnd=60 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=61 CharacterOffsetEnd=64 PartOfSpeech=DT] [Text=punk CharacterOffsetBegin=65 CharacterOffsetEnd=69 PartOfSpeech=NN] [Text=movement CharacterOffsetBegin=70 CharacterOffsetEnd=78 PartOfSpeech=NN] [Text=in CharacterOffsetBegin=79 CharacterOffsetEnd=81 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=82 CharacterOffsetEnd=85 PartOfSpeech=DT] [Text=LA CharacterOffsetBegin=86 CharacterOffsetEnd=88 PartOfSpeech=NNP] [Text=in CharacterOffsetBegin=89 CharacterOffsetEnd=91 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=92 CharacterOffsetEnd=95 PartOfSpeech=DT] [Text=early CharacterOffsetBegin=96 CharacterOffsetEnd=101 PartOfSpeech=JJ] [Text=80 CharacterOffsetBegin=102 CharacterOffsetEnd=104 PartOfSpeech=CD] [Text='s CharacterOffsetBegin=104 CharacterOffsetEnd=106 PartOfSpeech=POS] [Text=. CharacterOffsetBegin=106 CharacterOffsetEnd=107 PartOfSpeech=.] 
root(ROOT-0, description-9)
det(One-2, The-1)
num(description-9, One-2)
cc(One-2, and-3)
det(Only!The-5, the-4)
conj(One-2, Only!The-5)
advmod(good-8, only-6)
advmod(good-8, really-7)
amod(description-9, good-8)
prep(description-9, of-10)
det(movement-13, the-11)
nn(movement-13, punk-12)
pobj(of-10, movement-13)
prep(movement-13, in-14)
det(LA-16, the-15)
pobj(in-14, LA-16)
prep(description-9, in-17)
det(80-20, the-18)
amod(80-20, early-19)
pobj(in-17, 80-20)
possessive(80-20, 's-21)
punct(description-9, .-22)

Sentence #2 (27 tokens):
Also, the definitive documentary about legendary bands like the Black Flag and the X.
    Mainstream Americans' repugnant views about this film are absolutely hilarious!
[Text=Also CharacterOffsetBegin=112 CharacterOffsetEnd=116 PartOfSpeech=RB] [Text=, CharacterOffsetBegin=116 CharacterOffsetEnd=117 PartOfSpeech=,] [Text=the CharacterOffsetBegin=118 CharacterOffsetEnd=121 PartOfSpeech=DT] [Text=definitive CharacterOffsetBegin=122 CharacterOffsetEnd=132 PartOfSpeech=JJ] [Text=documentary CharacterOffsetBegin=133 CharacterOffsetEnd=144 PartOfSpeech=NN] [Text=about CharacterOffsetBegin=145 CharacterOffsetEnd=150 PartOfSpeech=IN] [Text=legendary CharacterOffsetBegin=151 CharacterOffsetEnd=160 PartOfSpeech=JJ] [Text=bands CharacterOffsetBegin=161 CharacterOffsetEnd=166 PartOfSpeech=NNS] [Text=like CharacterOffsetBegin=167 CharacterOffsetEnd=171 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=172 CharacterOffsetEnd=175 PartOfSpeech=DT] [Text=Black CharacterOffsetBegin=176 CharacterOffsetEnd=181 PartOfSpeech=JJ] [Text=Flag CharacterOffsetBegin=182 CharacterOffsetEnd=186 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=187 CharacterOffsetEnd=190 PartOfSpeech=CC] [Text=the CharacterOffsetBegin=191 CharacterOffsetEnd=194 PartOfSpeech=DT] [Text=X. CharacterOffsetBegin=195 CharacterOffsetEnd=197 PartOfSpeech=NNP] [Text=Mainstream CharacterOffsetBegin=202 CharacterOffsetEnd=212 PartOfSpeech=NNP] [Text=Americans CharacterOffsetBegin=213 CharacterOffsetEnd=222 PartOfSpeech=NNPS] [Text=' CharacterOffsetBegin=222 CharacterOffsetEnd=223 PartOfSpeech=POS] [Text=repugnant CharacterOffsetBegin=224 CharacterOffsetEnd=233 PartOfSpeech=JJ] [Text=views CharacterOffsetBegin=234 CharacterOffsetEnd=239 PartOfSpeech=NNS] [Text=about CharacterOffsetBegin=240 CharacterOffsetEnd=245 PartOfSpeech=IN] [Text=this CharacterOffsetBegin=246 CharacterOffsetEnd=250 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=251 CharacterOffsetEnd=255 PartOfSpeech=NN] [Text=are CharacterOffsetBegin=256 CharacterOffsetEnd=259 PartOfSpeech=VBP] [Text=absolutely CharacterOffsetBegin=260 CharacterOffsetEnd=270 PartOfSpeech=RB] [Text=hilarious CharacterOffsetBegin=271 CharacterOffsetEnd=280 PartOfSpeech=JJ] [Text=! CharacterOffsetBegin=280 CharacterOffsetEnd=281 PartOfSpeech=.] 
root(ROOT-0, hilarious-26)
advmod(hilarious-26, Also-1)
punct(hilarious-26, ,-2)
det(documentary-5, the-3)
amod(documentary-5, definitive-4)
nsubj(hilarious-26, documentary-5)
prep(documentary-5, about-6)
amod(bands-8, legendary-7)
pobj(about-6, bands-8)
prep(bands-8, like-9)
det(Flag-12, the-10)
amod(Flag-12, Black-11)
pobj(like-9, Flag-12)
cc(Flag-12, and-13)
det(Americans-17, the-14)
nn(Americans-17, X.-15)
nn(Americans-17, Mainstream-16)
poss(views-20, Americans-17)
possessive(Americans-17, '-18)
amod(views-20, repugnant-19)
conj(Flag-12, views-20)
prep(views-20, about-21)
det(film-23, this-22)
pobj(about-21, film-23)
cop(hilarious-26, are-24)
advmod(hilarious-26, absolutely-25)
punct(hilarious-26, !-27)

Sentence #3 (21 tokens):
How can music be SO diversive in a country of supposed liberty...even 20 years after...find out!
[Text=How CharacterOffsetBegin=286 CharacterOffsetEnd=289 PartOfSpeech=WRB] [Text=can CharacterOffsetBegin=290 CharacterOffsetEnd=293 PartOfSpeech=MD] [Text=music CharacterOffsetBegin=294 CharacterOffsetEnd=299 PartOfSpeech=NN] [Text=be CharacterOffsetBegin=300 CharacterOffsetEnd=302 PartOfSpeech=VB] [Text=SO CharacterOffsetBegin=303 CharacterOffsetEnd=305 PartOfSpeech=RB] [Text=diversive CharacterOffsetBegin=306 CharacterOffsetEnd=315 PartOfSpeech=JJ] [Text=in CharacterOffsetBegin=316 CharacterOffsetEnd=318 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=319 CharacterOffsetEnd=320 PartOfSpeech=DT] [Text=country CharacterOffsetBegin=321 CharacterOffsetEnd=328 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=329 CharacterOffsetEnd=331 PartOfSpeech=IN] [Text=supposed CharacterOffsetBegin=332 CharacterOffsetEnd=340 PartOfSpeech=VBN] [Text=liberty CharacterOffsetBegin=341 CharacterOffsetEnd=348 PartOfSpeech=NN] [Text=... CharacterOffsetBegin=348 CharacterOffsetEnd=351 PartOfSpeech=:] [Text=even CharacterOffsetBegin=351 CharacterOffsetEnd=355 PartOfSpeech=RB] [Text=20 CharacterOffsetBegin=356 CharacterOffsetEnd=358 PartOfSpeech=CD] [Text=years CharacterOffsetBegin=359 CharacterOffsetEnd=364 PartOfSpeech=NNS] [Text=after CharacterOffsetBegin=365 CharacterOffsetEnd=370 PartOfSpeech=IN] [Text=... CharacterOffsetBegin=370 CharacterOffsetEnd=373 PartOfSpeech=:] [Text=find CharacterOffsetBegin=373 CharacterOffsetEnd=377 PartOfSpeech=VB] [Text=out CharacterOffsetBegin=378 CharacterOffsetEnd=381 PartOfSpeech=RP] [Text=! CharacterOffsetBegin=381 CharacterOffsetEnd=382 PartOfSpeech=.] 
root(ROOT-0, diversive-6)
advmod(diversive-6, How-1)
aux(diversive-6, can-2)
nsubj(diversive-6, music-3)
cop(diversive-6, be-4)
advmod(diversive-6, SO-5)
prep(diversive-6, in-7)
det(country-9, a-8)
pobj(in-7, country-9)
prep(country-9, of-10)
amod(liberty-12, supposed-11)
pobj(of-10, liberty-12)
punct(diversive-6, ...-13)
quantmod(20-15, even-14)
num(years-16, 20-15)
npadvmod(after-17, years-16)
dep(diversive-6, after-17)
punct(diversive-6, ...-18)
dep(diversive-6, find-19)
prt(find-19, out-20)
punct(diversive-6, !-21)

